runtime/pangolin/tests/tests.rs

```diff
@@ -1,7 +1,10 @@
 pub mod mock;
 
+#[cfg(test)]
```

Comment:
Please sync this changes to other runtimes.

Reply:
Sorry, I forgot that this is an external test. Removed.